dali_tf_plugin/MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include Acknowledgements.txt LICENSE COPYRIGHT
-recursive-include nvidia/dali *.cc
+include *.cc
+include *.h
```

Comment:
Why we need to include *.h as well now?

Reply:
Because of `tfallocator.h`. It works now because including `tfallocator.h` is disabled but otherwise we will get an error